adventurer/templates/home.html

```diff
@@ -1,17 +1,9 @@
-<!-- wp:group {"style":{"spacing":{"blockGap":"0"}},"layout":{"type":"default"}} -->
-<div class="wp-block-group">
-    <!-- wp:pattern {"slug":"adventurer/header-with-cover"} /-->
-
-    <!-- wp:spacer {"height":"60px"} -->
-    <div style="height:60px" aria-hidden="true" class="wp-block-spacer"></div>
-    <!-- /wp:spacer -->
+<!-- wp:group {"style":{"spacing":{"blockGap":"0"}},"backgroundColor":"tertiary","layout":{"type":"default"}} -->
+<div class="wp-block-group has-tertiary-background-color has-background">
+    <!-- wp:template-part {"slug":"secondary-header"} /-->
 
     <!-- wp:post-content {"lock":{"move":false,"remove":false},"layout":{"type":"constrained","contentSize":"1100px"}} /-->
```

Comment:
What's the goal of using the post-content block in the home template? 

Right now if I have set to latest posts, it is displaying the content of the most recent post. Asking because I would imagine in most cases the view will differ quite a bit from the screenshot.

Reply:
Right! A big part of the screenshot is the contents of the List pattern.

@beafialho, should the List pattern be used in the home template, so the home page looks like the screenshot? At the moment, it's using the content of the most recent post (or sticky post). Or should this be a query block?